bindata/network/multus/003-dhcp-daemon.yaml

```diff
@@ -52,11 +52,19 @@ spec:
           mountPath: /host/run/cni
         - name: procpath
           mountPath: /host/proc
+        - name: netnspath
+          mountPath: /host/var/run/netns
+          mountPropagation: Bidirectional
```

Comment:
Just an FYI that this was required otherwise I was getting complaints about wrong fs magic.

Reply:
Bidirectional or HostToContainer could be okey. see: https://github.com/openshift/cluster-network-operator/pull/579/files

Reply:
Yes, please make it `HostToContainer` if that works. More securer.